charts/sonarqube/README.md

```diff
@@ -409,8 +409,7 @@ The following table lists the configurable parameters of the SonarQube chart and
 | Parameter                    | Description                                                   | Default |
 |------------------------------|---------------------------------------------------------------| ------- |
 | `tests.enabled`              | Flag that allows tests to be excluded from the generated yaml | `true` |
-| `tests.image`                | Change test container image                                   | `bitnami/minideb-extras` |
-| `tests.initContainers.image` | Change test init container image                              | `bats/bats:1.2.1` |
+| `tests.image`                | Change test container image                                   | `.Values.image.repository:.Values.image.tag` |
```

Comment:
I would mention `""` as the default value. It reflects what we have in the values file and people set this value only if they want to change from the SQ image...